src/object/floating_text.cpp

```diff
@@ -37,7 +37,7 @@ FloatingText::FloatingText(const Vector& pos, int score) :
 {
   timer.start(.1f);
 
-  // turn int into a string
+  // Turn int into a string.
   char str[10];
   snprintf(str, 10, "%d", score);
   text = str;
```

Comment:
Not related to spelling or comments, but `std::to_string` could be used here instead, and directly in the member initialization list.

Reply:
That's probably leftover from times before C++14 where `std::to_string` wasn't available.